guide/case-studies/blixt-wallet.md

```diff
@@ -0,0 +1,173 @@
+- **Prototype:** The re-designed screenstates were then used to create a clickable prototype to test out the user journey we had in mind.
+- **Test script:** We then created a pre-screening questionnaire as well as a test script which we would use during the testing process.
+- **Testing:** The prototype was then tested on 4 people. During the usability test, they clicked through the prototype and we observed how they interacted with it.
+
+After watching the users testing the wallet we made some observations. Those observations were then grouped into different parts of the user flow. All of the data from the tests can be found [here](https://docs.google.com/document/d/1NuYAQX2c_AAnHq6VPMXrg3IxuzL6Pa517SY1Jx6wAs8/edit?usp=sharing). (We have not included the names of the testers to protect their privacy). 
+
+## Here’s what we learned
+
+{% include picture.html
+   image = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-usability-testing-data-overview.png"
+   retina = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-usability-testing-data-overview@2x.png"
+   mobile = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-usability-testing-data-overview-mobile.png"
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-usability-testing-data-overview-mobile@2x.png"
+   alt-text = "Chart depicting an overview of the data from the usability tests, showing user confusion about usernames, opening channels, the copywriting, and how to fund the wallet by copying a QR code."
+   width = 800
+   height = 751
+%}
+
+- Users new to lightning are missing a crucial step, the why. They often lack understanding on why they need to open a channel before being presented with a screen to open one. Once this is clear it becomes easy for a user to understand that this step is necessary before transacting.
+*For more information on channels, see the [Technology Primer]({{'/guide/getting-started/technology-primer/#what-is-a-payment-channel' | relative_url}}) and [Lightning Liquidity]({{'/guide/how-it-works/liquidity/' | relative_url}}) section.*
+- Explaining to users how long it takes to open a channel sets a clear expectation around the process. This would be best done early on during the onboarding journey.
+
+Opening channels is a process unique to lightning wallets and an essential first step during the onboarding process. For users brand-new to lightning, opening a channel in itself is something that is new and unique to them. It is then helpful to explain to users why a channel should be opened before presenting them with the channel opening screen An example of the text could read:
+
+### Explaining the concept of opening a channel to a user that has never used a lightning wallet
+
+“First things first, to use your lightning wallet, you’ll need to set up a channel. Think of a channel as your electricity connection to the lightning network. By opening a channel you are able to make transactions.
```

Comment:
You could use the [definition list](https://deploy-preview-937--bitcoin-design-site.netlify.app/guide/contribute/formatting/#definition-list) format for these. It shows a piece of copy on the left, and an explainer on the right.

Reply:
I'm not sure what specifically you are referring to here. Are you referring to the image below with the black UI screen and all the text describing it? If so, Mo and I don't feel that definition list is a good way of presenting that information. If you're concerned about the a11y and SEO, I can come up with some other ways of formatting the information. We were hoping to tackle in a future PR< but can go ahead and do now if this is a blocker for yoyu.

Or where you referring to something else?